third_party/amd/backend/compiler.py

```diff
@@ -255,6 +255,8 @@ def make_ttgir(mod, metadata, options):
         passes.ttgpuir.add_reduce_data_duplication(pm)
         if amd.has_matrix_core_feature(options.arch):
             amd.passes.ttgpuir.add_reorder_instructions(pm)
+            if os.getenv("TRITON_HIP_USE_BLOCK_PINGPONG") == "1" and options.num_stages == 2:
```

Comment:
Acutally do `getenv(..., "0") == "1"` like other places to be explicit. You also need to add this to `GetEnv.cpp` to make sure changing it would flush cache.

Reply:
Addressed with 90e2bd1